test/unit/com/unifina/service/BalanceServiceSpec.groovy

```diff
@@ -0,0 +1,50 @@
+package com.unifina.service
+
+import com.unifina.api.ApiException
+import grails.test.mixin.TestFor
+import org.web3j.exceptions.MessageDecodingException
+import spock.lang.Specification
+
+import java.util.concurrent.ExecutionException
+
+@TestFor(BalanceService)
+class BalanceServiceSpec extends Specification {
+	def address = "0x494531425508c4Bc95E522b24fd571461583E916"
+	def setup() {
+		service.web3 = Mock(Web3Balance)
+    }
```

Comment:
Indentation off

Reply:
I think that is just GitHub UI. Actual code looks good to me!